Maker-Agents-Suit-Module/OSMAgent/src/priceUpdateCheck.ts

```diff
@@ -0,0 +1,76 @@
+import BigNumber from "bignumber.js";
+import {
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+} from "forta-agent";
+
+import TimeTracking from "./TimeTracking";
+import {
+  provideFunctionCallsDetectorAgent,
+  FindingGenerator,
+} from "@nethermindeth/general-agents-module";
+
+const time = new TimeTracking();
+const address = "0x2417c2762ec12f2696f62cfa5492953b9467dc81";
+export const functionSignature = "function poke()";
```

Comment:
The signature doesn't include the word `function`.
```suggestion
export const functionSignature = "poke()";
```